spec/controllers/person/remarks_controller_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+require 'spec_helper'
+
+describe Person::RemarksController do
+  context 'GET index' do
+    let(:person) { people(:mitglied) }
+    before { sign_in(person) }
+
+    subject(:request) do
```

Comment:
schau dir mal bestehende controller tests an, ich würde hier gänzlich auf subject verzichten und den request direkt in den it blöcken anwenden